<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APIEdit</class>
 <widget class="QDialog" name="APIEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>426</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>API Edotor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="LineAPIUrl">
       <property name="placeholderText">
        <string>Entre the API URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TypeURL">
       <item>
        <property name="text">
         <string>Get</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Post</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonTest">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="LineName">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageGet">
      <property name="minimumSize">
       <size>
        <width>262</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="GetResult">
         <property name="title">
          <string>Result</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="CupyToClipboard">
            <property name="text">
             <string>Copy to clipboared</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Window">
            <property name="text">
             <string>Window</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="SaveToFile">
            <property name="text">
             <string>Save to file</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="LineFilePath">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="ButtonSelectFile_Get">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="Fileter">
              <property name="text">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="FilterListType">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>White</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Black</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="FilterType">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Index</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Key</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="LineFileter">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Single value or list of values</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="GetData">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QCheckBox" name="GetFromClipboad_Get">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Get from clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineKey_Get">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Key</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineDataJson">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>2048</number>
            </property>
            <property name="placeholderText">
             <string>Data format Json</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pagePoss">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="PostData">
         <property name="title">
          <string>Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QCheckBox" name="GetFormClipboard_Post">
            <property name="text">
             <string>Get from clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LineKey_Post">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Key</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="LineData">
              <property name="placeholderText">
               <string>Data or file path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="ButtonSelectFile_Post">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>200</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LineAPIUrl</tabstop>
  <tabstop>TypeURL</tabstop>
  <tabstop>CupyToClipboard</tabstop>
  <tabstop>LineFilePath</tabstop>
  <tabstop>ButtonSelectFile_Get</tabstop>
  <tabstop>Fileter</tabstop>
  <tabstop>FilterType</tabstop>
  <tabstop>LineFileter</tabstop>
  <tabstop>ButtonSelectFile_Post</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>GetFormClipboard_Post</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineKey_Post</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>58</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>APIEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveToFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>ButtonSelectFile_Get</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineAPIUrl</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LineName</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveToFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineFilePath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fileter</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineFileter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>APIEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>10</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fileter</sender>
   <signal>toggled(bool)</signal>
   <receiver>FilterType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GetFromClipboad_Get</sender>
   <signal>toggled(bool)</signal>
   <receiver>LineKey_Get</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fileter</sender>
   <signal>toggled(bool)</signal>
   <receiver>FilterListType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
